spec/lib/dor/text_extraction/cocina_updater_spec.rb

```diff
@@ -380,7 +388,15 @@
                 hasMimeType: 'application/xml',
                 use: 'transcription',
                 sdrGeneratedText: sdr_generated_text,
-                correctedForAccessibility: corrected_for_accessibility
+                correctedForAccessibility: corrected_for_accessibility,
+                hasMessageDigests: [{
+                  type: 'md5',
+                  digest: 'bogus_md5'
+                },
+                                    {
+                                      type: 'sha1',
+                                      digest: 'bogus_sha1'
+                                    }]
               }
```

Comment:
this new mocked data lets us test for the replacement of the existing file below